airbyte-commons/src/main/java/io/airbyte/commons/text/Sqls.java

```diff
@@ -22,34 +22,20 @@
  * SOFTWARE.
  */
 
-package io.airbyte.scheduler;
+package io.airbyte.commons.text;
 
-import com.google.common.base.Preconditions;
-import io.airbyte.config.JobConfig;
+import java.util.stream.Collectors;
+import java.util.stream.StreamSupport;
 
-public class ScopeHelper {
+public class Sqls {
 
-  private static final String SCOPE_DELIMITER = ":";
-
-  public static String getScopePrefix(JobConfig.ConfigType configType) {
-    return configType.value();
-  }
-
-  public static String createScope(JobConfig.ConfigType configType, String configId) {
-    Preconditions.checkNotNull(configType);
-    Preconditions.checkNotNull(configId);
-    return getScopePrefix(configType) + SCOPE_DELIMITER + configId;
+  public static <T extends Enum<T>> String toSqlName(final T value) {
+    return value.name().toLowerCase();
   }
 
-  public static String getConfigId(String scope) {
-    Preconditions.checkNotNull(scope);
-
-    final String[] split = scope.split(SCOPE_DELIMITER);
-    if (split.length <= 1) {
-      return "";
-    } else {
-      return split[1];
-    }
+  public static <T extends Enum<T>> String toSqlInFragment(final Iterable<T> values) {
```

Comment:
not sure i understand this method name. `iterableToSql` maybe?

Reply:
I want to say that it generate the fragment of sql that you put inside the `IN` clause

Reply:
ooh. got it. okay, name is maybe okay then. i'd add a java doc comment. it's going to be hard to fully convey what you mean in a method name.